### PR TITLE
PAYARA-2123 admin console view log files fix

### PR DESCRIPTION
**Pull request by MattGill98**
- Fixed REST response formats for log viewing endpoints.
- Restored view-log page functionality
- Restored view-raw-log page functionality

### CODE REVIEW

**Comment by MattGill98** [comment]
jenkins quick test payara 5 please

**Comment by MattGill98** [comment]
jenkins quick test payara 5 please

File: nucleus/core/logging/src/main/java/com/sun/enterprise/server/logging/logviewer/backend/LogFilter.java [hunk 1]
```diff
@@ -224,9 +225,9 @@ public AttributeList getLogRecordsUsingQuery(
         }
     }
 
-    public Vector getInstanceLogFileNames(String instanceName) {
+    public List<String> getInstanceLogFileNames(String instanceName) {
```
Comment (arjantijms) [inline nucleus/core/logging/src/main/java/com/sun/enterprise/server/logging/logviewer/backend/LogFilter.java]: Thank you for that!

File: nucleus/core/logging/src/main/java/com/sun/enterprise/server/logging/logviewer/backend/LogFilterForInstance.java [hunk 2]
```diff
@@ -98,7 +100,7 @@ public File downloadGivenInstanceLogFile(ServiceLocator habitat, Server targetSe
             String loggingDir = getLoggingDirectoryForNode(instanceLogFileName, node, sNode, instanceName);
 
             try {
-                Vector instanceLogFileNames = sftpClient.ls(loggingDir);
+                List instanceLogFileNames = sftpClient.ls(loggingDir);
```
Comment (arjantijms) [inline nucleus/core/logging/src/main/java/com/sun/enterprise/server/logging/logviewer/backend/LogFilterForInstance.java]: Can probably add the right generic param here

Comment (MattGill98) [reply nucleus/core/logging/src/main/java/com/sun/enterprise/server/logging/logviewer/backend/LogFilterForInstance.java]: Cheers, good spot! Made that change now

**Comment by arjantijms** [comment]
jenkins quick test payara 5 please

**Comment by arjantijms** [comment]
jenkins quick test payara 5 please

**Pull request merged**